lib/exposes.js

```diff
@@ -335,6 +335,12 @@ class Cover extends Base {
         this.features.push(new Numeric('tilt', access.ALL).withValueMin(0).withValueMax(100).withDescription('Tilt of this cover'));
         return this;
     }
+
+    withRunning() {
```

Comment:
I'm sorry for not being clear, what I mean was adding this to `exposes` like `exposes.binary('running', ea.STATE, true, false)` (so not making it part of the feature list)

Reply:
In the first time I tried this, but can`t figure out how to check/find exposed value in this place: https://github.com/Koenkk/zigbee2mqtt/blob/7a2ddf24d32006bc05283c5ceb681dc9871b0347/lib/extension/homeassistant.ts#L380-L427